codec-dns/src/test/java/io/netty/handler/codec/dns/DefaultDnsRecordDecoderTest.java

```diff
@@ -17,30 +17,33 @@
 
 import io.netty.buffer.ByteBuf;
 import io.netty.buffer.Unpooled;
+import io.netty.channel.socket.DatagramPacket;
 import io.netty.util.internal.StringUtil;
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.nio.CharBuffer;
+
 public class DefaultDnsRecordDecoderTest {
 
     @Test
     public void testDecodeName() {
-        testDecodeName("netty.io.", Unpooled.wrappedBuffer(new byte[] {
+        testDecodeName("netty.io.", Unpooled.wrappedBuffer(new byte[]{
```

Comment:
Nooo.. Could you configure your IDE's formatter first? http://netty.io/wiki/setting-up-development-environment.html